addons/binding/org.openhab.binding.yeelight/README.md

```diff
@@ -60,11 +60,11 @@ Channel | openHAB Type | Description
 Things:
 
 ```
-Thing yeelight:stripe:0x000000000321a1bc [ deviceId="0x000000000321a1bc" ]
+Thing yeelight:stripe:1 [ deviceId="0x000000000321a1bc" ]
```

Comment:
Just for clarification: Is the "1" the UID of the thing? Because the old binding worked with the deviceId there and called the lib with something like `DeviceManager.getInstance().doAction(channelUID.getThingUID().getId(), ...` and I wonder where it gets the Id from/whether my changed implementation is correct.

Also currently the DiscoveryService sets the deviceId as the UID, since I guess it needs to be unique. So a GUI configuration will result with the deviceId in that place.

Reply:
Alright, so the UID are actually all segments together? Does `channelUID.getThingUID().getId()` only return the last part of the UID or all segments? Because I currently just replaced it with the last segment of it.

Reply:
Correct.